### DISCUSSION TITLE
Backticks aren't properly handled in discussions

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#133090] [Mobile] [answered]
GitHub Mobile beta for Android

1. Open Chrome
2. Visit https://github.com/community/community/discussions/133089
3. See backticks parsed as markdown ![Screenshot_20240722-083006.png](https://github.com/user-attachments/assets/f1384901-e9f9-4514-a1db-4e5d95b3eac4)

4. Open the same item in GitHub Mobile
5. See backticks rendered as raw text ![Screenshot_20240722-082812.png](https://github.com/user-attachments/assets/b76a7512-aefa-46a3-a49b-6b78c068fd57)

**Comment by iskae** [comment, marked as answer]
Hi @jsoref 👋 

Thank you for reaching out ❤️ We already have an internal tracking issue for this feature 😊 Adding your request in there too and hopefully we can work on it soon 🙇 

Cheers!
